README.md

```diff
@@ -9,9 +24,32 @@ This library is in early development stages and will initially contain the relev
 from [piet-scene] but the intent is to be a generally useful set of primitives for vector
```

Comment:
piet-scene appears to be a dead link now.  I imagine the historical significance of this is no longer relevant and this entire line should probably be removed though it will leave the readme rather bare.